src/Layers/xrRenderDX10/dx10Texture.cpp

```diff
@@ -104,7 +104,7 @@ IC void Reduce(int& w, int& h, int& l, int& skip)
         h = 1;
 }
 
-IC void Reduce(UINT& w, UINT& h, int l, int skip)
+IC void Reduce(u32& w, u32& h, int l, int skip)
```

Comment:
Used with DirectX (D3DX) types.
```suggestion
IC void Reduce(UINT& w, UINT& h, int l, int skip)
```